### DISCUSSION TITLE
Add result to RPN stack multiple times

### DISCUSSION THREAD

**Discussion by jadav2323** [Qalculate/libqalculate#592] [Q&A] [answered]
I know I used to be able to enter a number to an RPN stack more than one time by hitting enter however many times I wanted to add the number.  Now I can only hit enter and add the number once, hitting enter more times does nothing.  I cannot find what setting has changed nor have I been able to find a solution by searching the help files or internet.  Using the GTK GUI

Thanks

**Comment by hanna-kn** [comment, marked as answer]
It seems as if this feature was unintentionally removed in version 4.1.1 when https://github.com/Qalculate/qalculate-gtk/issues/394 was fixed. 

The Enter keypad button, and the RPN copy button and keyboard shortcut (which can be assigned to any key combination) do however work as before.

 I've now fixed this (https://github.com/Qalculate/qalculate-gtk/commit/86952b018a69921ac6fa392bed01bba498c22480).
